example/all.yaml

```diff
@@ -3,6 +3,7 @@
 linter:
   rules:
     - always_declare_return_types
+    - always_depend_on_packages_you_use
```

Comment:
in general I think we're trending away from "always" and similar prefixes.  Bike shedding names...

* `direct_package_dependencies`

?

Reply:
`explicit_package_dependencies`

?

Reply:
Or `depend_on_referenced_packages`

Reply:
I am fine with any of these, I like `depend_on_referenced_packages` probably the most?

Reply:
I went with that name for now, LMK if you want to change it